core/client/fs/src/main/java/alluxio/client/file/cache/limiter/NoopWriteLimiter.java

```diff
@@ -0,0 +1,20 @@
+package alluxio.client.file.cache.limiter;
+
+import alluxio.conf.AlluxioConfiguration;
+
+/**
+ * Noop write limiter.
+ */
+public class NoopWriteLimiter implements WriteLimiter {
+  /**
+   * Constructor for NoopWriteLimiter.
+   * @param conf
+   */
+  public NoopWriteLimiter(AlluxioConfiguration conf) {
+  }
+
+  @Override
+  public boolean tryAcquire(int writeLength) {
+    return true;
```

Comment:
curiously why it is named tryAcquire? the name seems a bit confusing in terms of what it tries to achieve

Reply:
Yeah, I have the same feeling, naming is har, any idea? `tryWrite` or `isWritable` or `isThrottled`

Reply:
I'm thinking of something shouldThrottle? 

also, I just realized it could be useful to have page id as another parameter. e.g In the case that we want to throttle based on the temperature of the page, we can use page id to track it's temperature

Reply:
good idea!  how about just passing in the PageInfo? then we could get everything we need such as page_id, length, dir, and probably also the last modified time in the future